tnefparse/util.py

```diff
@@ -45,7 +45,7 @@ def systime(byte_arr, offset=0):
         return datetime.utcfromtimestamp((ft - EPOCH_AS_FILETIME) / HUNDREDS_OF_NANOSECONDS)
     except:  # noqa: E722
         microseconds = ft / 10
-        return (datetime(1601, 1, 1) + timedelta(microseconds=microseconds))
+        return datetime(1601, 1, 1) + timedelta(microseconds=microseconds)
```

Comment:
There is no coverage for this line. It would be great to have a test for it.

Reply:
Seems to be non-trivial to come up with a test case https://github.com/koodaamo/tnefparse/issues/92#issuecomment-735883589